.cirrus.yml

```diff
@@ -13,7 +13,7 @@ arm_docker_builder:
   env:
     CIRRUS_ARCH: arm64
     GITHUB_TOKEN: ENCRYPTED[98160948dd9e4613e0f7704958694e53fd7f021dd847938991e91c80e14ac5c91822aa9dff3fa34e52ec76f331beeeb5]
-  build_script: docker build -t ghcr.io/cheriot-platform/devcontainer:aarch64-latest .
+  build_script: docker build -t ghcr.io/cheriot-platform/devcontainer:aarch64-latest -t ghcr.io/cheriot-platform/devcontainer:$DATE_TAG .
```

Comment:
The date tag environment variable isn't set here.

Note that we need the environment variable to uniform here, and I'm not sure if there's a way of setting environment variables to a global.  I don't see anything in the predefined list that we could use.

Reply:
I though that's what pushing it into $CIRRUS_ENV would do